migration/sql-test-files/025-before-fix-feature-level.sql

```diff
@@ -0,0 +1 @@
+insert into users (id, cluster, email, email_private,feature_level) values ('a2ff2e0a-724c-4318-a8ef-38441f5205f6', 'test', 'migration-test-1025+preview@mail.com', false,'');
```

Comment:
what about a `nil` value in the `feature_level` column, too ? just to make sure we're not missing a case...

Reply:
good point!

Reply:
the column is set to have NOT NULL

Reply:
😂